shoes-core/spec/shoes/shared_examples/state.rb

```diff
@@ -1,7 +1,8 @@
 shared_examples_for "object with state" do
   let(:input_opts) { {state: "disabled"} }
 
-  it "should initialize" do
+  it "should initialize in the right state" do
+    expect_any_instance_of(Shoes.configuration.backend_class(described_class)).to receive(:enabled).with(false)
```

Comment:
Yep, indeed it was. I.e. testing the `after_initialize` bit. Proved its worth, i.e. The one in list_box didn't call super, hence did not call the `State` module method and hence it failed :)